hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/key/OMKeyRenameRequest.java

```diff
@@ -155,6 +156,9 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
       //Set modification time
       fromKeyValue.setModificationTime(renameKeyArgs.getModificationTime());
 
+      // Set the UpdateID to current transactionLogIndex
+      fromKeyValue.setUpdateID(transactionLogIndex);
```

Comment:
As this is like new key is getting created, so do we need to set objectID also here?

Reply:
I think either way should be ok. I thought since its the same key being renamed, we could have the same objectID. But I am ok changing it to current transactionLogIndex. Let me know what you think.

Reply:
As discussed offline, we can change this when we use them in finding a transaction is replay or not.